pkg/descheduler/strategies/duplicates.go

```diff
@@ -46,12 +55,43 @@ func RemoveDuplicatePods(ds *options.DeschedulerServer, strategy api.Descheduler
 // deleteDuplicatePods evicts the pod from node and returns the count of evicted pods.
 func deleteDuplicatePods(client clientset.Interface, policyGroupVersion string, nodes []*v1.Node, dryRun bool, nodepodCount utils.NodePodEvictedCount, maxPodsToEvict int, evictLocalStoragePods bool) int {
 	podsEvicted := 0
+	cepm := creatorEvictedPods{}
 	for _, node := range nodes {
 		klog.V(1).Infof("Processing node: %#v", node.Name)
 		dpm := ListDuplicatePodsOnANode(client, node, evictLocalStoragePods)
 		for creator, pods := range dpm {
+			if _, found := cepm[creator]; !found {
+				cepm[creator] = 0
+			}
+			status, err := getStatusForOwnerRef(client, creator)
+			if err != nil {
+				klog.Errorf("Error getting the number of replicas: %v", err)
+				continue
+			}
+			// bestRatio := float64(status.replicas) / float64(len(nodes))
+			bestRatio := (int(status.replicas) / len(nodes)) + 1
```

Comment:
Why `+1`?

Without +1 `bestRatio` says how much each pod of a RC/RS contributes to per node distribution. E.g. With 5 nodes and 10 replicas, each pod gives 0.5 pod/node (half a pod per each node).

Reply:
@ingvagabund was this a typo? 0.5 pod/node or 2 pod/node?

Reply:
0.5